maat-court-data-api/src/main/java/gov/uk/courtdata/hardship/validator/HardshipReviewValidationProcessor.java

```diff
@@ -0,0 +1,41 @@
+package gov.uk.courtdata.hardship.validator;
+
+import gov.uk.courtdata.exception.ValidationException;
+import gov.uk.courtdata.model.hardship.CreateHardshipReview;
+import gov.uk.courtdata.model.hardship.HardshipReview;
+import gov.uk.courtdata.model.hardship.UpdateHardshipReview;
+import gov.uk.courtdata.validator.HardshipReviewIdValidator;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class HardshipReviewValidationProcessor {
+
+    private final HardshipReviewIdValidator hardshipReviewIdValidator;
+    private final CreateHardshipReviewValidator createHardshipReviewValidator;
+    private final UpdateHardshipReviewValidator updateHardshipReviewValidator;
+
```

Comment:
I would have all hardship validators in one class like HardshipReviewValidator

Reply:
This is the pattern being followed in this project, so this will have to remain as is for consistency